### PR TITLE
feat(CLOUDDEV-806): added utils InstanceNetworkInterfaceByID and SecurityGroupListByIDs

### PR DESCRIPTION
**Pull request by DamirZinatullin**
https://tracker.yandex.ru/CLOUDDEV-806

### CODE REVIEW

**Review by Leonid-St — APPROVED** [review]

File: util/securitygroups.go [hunk 1]
```diff
@@ -37,3 +43,25 @@ func (s SecurityGroupRuleProtocol) StringList() []string {
...
 	return strings
 }
+
+func SecurityGroupListByIDs(ctx context.Context, client *edgecloud.Client, sgIDs []string) (sgs []edgecloud.SecurityGroup, err error) {
+	allSgs, _, err := client.SecurityGroups.List(ctx, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	allSGsMap := make(map[string]edgecloud.SecurityGroup)
```
Comment (alex2304) [inline util/securitygroups.go:53]: nit: `..., 0, len(allSgs)`

File: util/securitygroups.go [hunk 2]
```diff
@@ -37,3 +43,25 @@ func (s SecurityGroupRuleProtocol) StringList() []string {
 
 	return strings
 }
+
+func SecurityGroupListByIDs(ctx context.Context, client *edgecloud.Client, sgIDs []string) (sgs []edgecloud.SecurityGroup, err error) {
```
Comment (alex2304) [inline util/securitygroups.go:47]: nit: `targetSgIDs`

**Review by alex2304 — APPROVED** [review]

File: util/securitygroups.go (hunk 1 above)
Comment (DamirZinatullin) [reply util/securitygroups.go:53]: fixed

File: util/securitygroups.go (hunk 2 above)
Comment (DamirZinatullin) [reply util/securitygroups.go:47]: changed

**Pull request merged**